src/main/scala/cromwell/binding/Call.scala

```diff
@@ -34,6 +35,27 @@ case class Call(alias: Option[String], task: Task, inputMappings: Map[String, Wd
    * @return Map[String, WdlType] representing each task-local input
    */
   def unsatisfiedInputs: Map[String, WdlType] = task.inputs.filterNot { case (k, v) => inputMappings.contains(k) }
-  
+
   override def toString: String = s"[Call name=$name, task=$task]"
+
+  /** Find all calls upon which this call immediately depends, i.e. the result of this
+    * does not include transitive dependencies.  Currently this only works for member
+    * access expressions with a literal LHS, e.g.:
+    * 
+    * {{{
+    *   call cgrep {
+    *     input: in_file=ps.procs
+    *   }
+    * }}}
+    * 
+    */
+  def prerequisiteCalls(): Traversable[Call] = {
```

Comment:
There should not be duplicate calls in `prerequisiteCalls` since there shouldn't be duplicate calls anywhere.  I don't think this is the place for enforcing that invariant.

`findRunnableCalls` conceptually shouldn't require the return of `prerequisiteCalls` to deliver set semantics and shouldn't be required to pay the overhead for `equals` and `hashCode` invocations on each call to package them into a set.  Yes the current implementation of `findRunnableCalls` is doing a `toSet`, but hopefully that ugliness can be refactored away or at least kept confined to `findRunnableCalls`. :smile: